src/Microsoft.ML.Data/Evaluators/Metrics/ClusteringMetrics.cs

```diff
@@ -26,30 +26,31 @@ public sealed class ClusteringMetrics
         /// Note however, that this metric will only decrease if the number of clusters is increased,
         /// and in the extreme case (where each distinct example is its own cluster) it will be equal to zero.
         /// </summary>
-        public double AvgMinScore { get; }
+        /// <value>Distance is to the nearest centroid.</value>
+        public double AverageDistance { get; }
 
         /// <summary>
-        /// <a href="https://en.wikipedia.org/wiki/Davies%E2%80%93Bouldin_index">Davies-Bouldin Index</a>
-        /// DBI is a measure of the how much scatter is in the cluster and the cluster separation.
+        /// Davies-Bouldin Index is measure of the how much scatter is in the cluster and the cluster separation.
         /// </summary>
-        public double Dbi { get; }
+        /// <remarks><a href="https://en.wikipedia.org/wiki/Davies%E2%80%93Bouldin_index">Davies-Bouldin Index</a></remarks>
```

Comment:
>/// <remarks> [](start = 8, length = 13)

Remarks show up as a section in the documentation, so make sure the remarks are all full sentences.

Reply:
You mean with a full stop in the end?

---
In reply to: [260534630](https://github.com/dotnet/machinelearning/pull/2700#discussion_r260534630) [](ancestors = 260534630)